rs/sns/root/proto/ic_sns_root/pb/v1/root.proto

```diff
@@ -45,10 +45,6 @@ message SnsRootCanister {
   // True if the SNS is running in testflight mode. Then additional
   // controllers beyond SNS root are allowed when registering a dapp.
   bool testflight = 8;
-
-  // Set to `true` if the framework canisters' memory limit doesn't need to be changed
-  // TODO(NNS1-3286): remove
-  optional bool updated_framework_canisters_memory_limit = 9;
```

Comment:
This should be safe to delete, since proto will ignore the extra field when decoding